ui/src/components/ResourcesConfigTable.js

```diff
@@ -33,6 +34,12 @@ export const ResourcesConfigTable = ({
       title: "CPU Request",
       description: cpu_request,
     },
+    ...(cpu_limit !== undefined && cpu_limit !== "0" && cpu_limit !== "") ? [
```

Comment:
if user create with `cpu_limit=0`, the platform set it to `10`, i assume they will see 10 here?

Reply:
Nope they wouldn't, because when the platform controls the CPU limit, regardless of the value that gets automatically sets, it isn't saved in the DB. Hence when the UI retrieves the config values, the `cpu_limit` value would be empty.